gears/flywheel/presidio-analyzer.json

```diff
@@ -0,0 +1,51 @@
+{
+  "author": "Flywheel",
+  "cite": "https://microsoft.github.io/presidio/",
+  "command": "python run.py",
+  "config": {
+    "debug": {
+      "default": false,
+      "description": "Log debug messages",
+      "type": "boolean"
+    }
+  },
+  "custom": {
+    "flywheel": {
+      "suite": "PHI-Detection",
+      "show-job": true
+    },
+    "gear-builder": {
+      "category": "classifier",
+      "image": "flywheel/presidio-analyzer:0.1.0"
+    }
+  },
+  "description": "This gear utilizes Mircosoft's Presidio SDK which utilizes Natural Language Processing techiniques like Regex and NER to de-identify PII for subject and paitent data protection.",
+  "environment": {
+    "PATH": "/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+    "LANG": "C.UTF-8",
+    "GPG_KEY": "E3FF2839C048B25C084DEBE9B26995E310250568",
+    "PYTHON_VERSION": "3.8.16",
+    "PYTHON_PIP_VERSION": "22.0.4",
+    "PYTHON_SETUPTOOLS_VERSION": "57.5.0",
+    "PYTHON_GET_PIP_URL": "https://github.com/pypa/get-pip/raw/1a96dc5acd0303c4700e02655aefd3bc68c78958/public/get-pip.py",
+    "PYTHON_GET_PIP_SHA256": "d1d09b0f9e745610657a528689ba3ea44a73bd19c60f4c954271b790c71c2653",
+    "FLYWHEEL": "/flywheel/v0"
+  },
+  "inputs": {
+    "input_file": {
+      "base": "file",
+      "description": "A text, csv, tsv, or json file"
```

Comment:
Nit: Can you add the correspond `enum` value for those file type ?

Reply:
enum values added